scripts/change-password.sh

```diff
@@ -1,11 +1,17 @@
 #!/bin/bash
 
-curl --include --request PATCH http://localhost:3000/change-password/$ID \
-  --header "Authorization: Token token=$TOKEN" \
+API="http://localhost:4741"
```

Comment:
I like it better. I don't know why. Convince me otherwise?
I too often see people paste a curl script that looks like the following without passing what TOKEN and ID first...
```bash
curl --include --request PATCH http://localhost:3000/change-password/$ID \
  --header "Authorization: Token token=$TOKEN"
```